manifests/deploy.yaml

```diff
@@ -184,6 +184,8 @@ spec:
                   type: array
                 privateRegistry:
                   type: string
+                namespaceOverride:
```

Comment:
This field also can be removed. The reason is the same as above. And I will update this deploy file later.

Reply:
done